### ISSUE TITLE
incompatible_load_proto_rules_from_bzl: load the Protobuf rules from @rules_proto

### ISSUE THREAD

**Issue by Yannic** [bazelbuild/bazel#8922]
Flag: `--incompatible_load_proto_rules_from_bzl`
Available since: 0.29
Will be flipped in: TBD
Tracking issue: #8891

`proto_library` and `proto_lang_toolchain` have to be loaded from [@rules_proto](https://github.com/bazelbuild/rules_proto).

For example, if you are using`proto_library`, add the following load statement to your BUILD/bzl file.
```python
load("@rules_proto//proto:defs.bzl", "proto_library")
```

You can use the following buildifier command to automate the changes to your BUILD/bzl files:
```
buildifier --lint=fix --warnings=native-proto <path/to/BUILD>
```

You will have to add `proto_library` to your WORKSPACE file as well:

```python
load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
http_archive(
    name = "rules_proto",
    sha256 = "602e7161d9195e50246177e7c55b2f39950a9cf7366f74ed5f22fd45750cd208",
    strip_prefix = "rules_proto-97d8af4dc474595af3900dd85cb3a29ad28cc313",
    urls = [
        "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/97d8af4dc474595af3900dd85cb3a29ad28cc313.tar.gz",
        "https://github.com/bazelbuild/rules_proto/archive/97d8af4dc474595af3900dd85cb3a29ad28cc313.tar.gz",
    ],
)
load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_proto_toolchains")
rules_proto_dependencies()
rules_proto_toolchains()
```

**Comment by Yannic** [comment]
@iirina Can you add the labels `breaking-change-1.0`, `incompatible-change`, `migration-ready`, and `migration-0.29` per https://docs.bazel.build/versions/0.28.0/backward-compatibility.html#github-labels? Thanks!

**Comment by iirina** [comment]
@Yannic done! Thanks!

**Comment by dslomov** [comment]
Will the migration tooling be available to do the migration? (based on buildifier)

**Comment by Yannic** [comment]
Yes, a buildifier fix has been added in https://github.com/bazelbuild/buildtools/pull/674

**Comment by limdor** [comment]
What is the status of this flag, it is saying will be flipped TBD but breaking change 5.0. Is it flipped in Bazel 5?

**Comment by lberki** [comment]
@philwo I assume we are not flipping this in 5.0 since we decided to go with builtin Starlark rules instead for the time being?

**Comment by Yannic** [comment]
I don't think it's ready for flipping in 5.0 (would like to do it eventually though)

**Comment by comius** [comment]
Absolutely, not flipping in 5.0. I removed the label on the issue.

**Comment by meisterT** [comment]
This issue is obsolete. The `--incompatible_load_proto_rules_from_bzl` flag was deprecated and converted to a no-op in commit 5ab2d2e02e201835fcb0272dcebfd120c3b79f10, and tracking issue #8891 was closed.
